### ISSUE TITLE
'Cannot declare class craft\behaviors\ContentBehavior' error on POST

### ISSUE THREAD

**Issue by howells** [craftcms/cms#5361]
### Description

I've found that occasionally, when I made post requests to my CraftQL endpoint, I get a PHP Compile Error:

```
name: "PHP Compile Error"
message: "Cannot declare class craft\behaviors\ContentBehavior, because the name is already in use"
code: 64
type: "yii\base\ErrorException"
file: "/.../storage/runtime/compiled_classes/ContentBehavior.php"
line: 346
```

I've raise it in Craft since it doesn't seem to be specific to CraftQL itself. This is on my local development environment, using MAMP.

### Steps to reproduce

Make multiple POST requests to a CraftQL end point. It's very hard to reliably replicate, but at least 1 out of approx 10 requests fail.

### Additional info

- Craft version: 3.3.18.3
- PHP version: 7.3
- Database driver & version: Mysql
- Plugins & versions: CraftQL 1.3.4

**Comment by brandonkelly** [comment]
Can you find that error in your storage/logs/ folder, and post the stack trace?

**Comment by howells** [comment]
Sure, here you go. Let me know if you need anything else from me.

```
2019-12-20 23:23:34 [-][-][-][error][yii\base\ErrorException:64] yii\base\ErrorException: Cannot declare class craft\behaviors\ContentBehavior, because the name is already in use in /Users/danielhowells/Sites/********/api/storage/runtime/compiled_classes/ContentBehavior.php:346
Stack trace:
#0 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/base/ErrorHandler.php(0): craft\web\ErrorHandler->handleFatalError()
#1 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/Craft.php(204): Craft::_loadFieldAttributesFile()
#2 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/Craft.php(328): Craft::_autoloadContentBehavior()
#3 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/Craft.php(186): Craft::_autoloadElementQueryBehavior()
#4 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/di/Container.php(447): Craft::autoload()
#5 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/di/Container.php(447): ::spl_autoload_call()
#6 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/di/Container.php(447): ReflectionClass->__construct()
#7 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/di/Container.php(374): yii\di\Container->getDependencies()
#8 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/di/Container.php(159): yii\di\Container->build()
#9 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/BaseYii.php(345): yii\di\Container->get()
#10 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/base/Component.php(750): yii\BaseYii::createObject()
#11 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/base/Component.php(734): craft\elements\db\EntryQuery->attachBehaviorInternal()
#12 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/base/Component.php(465): craft\elements\db\EntryQuery->ensureBehaviors()
#13 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/db/Query.php(54): craft\elements\db\EntryQuery->hasEventHandlers()
#14 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/elements/db/EntryQuery.php(243): craft\elements\db\EntryQuery->init()
#15 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\elements\db\EntryQuery->init()
#16 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/elements/db/ElementQuery.php(488): craft\elements\db\EntryQuery->__construct()
#17 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/elements/db/EntryQuery.php(211): craft\elements\db\EntryQuery->__construct()
#18 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/elements/Entry.php(157): craft\elements\db\EntryQuery->__construct()
#19 /Users/danielhowells/Sites/********/api/vendor/rias/craft-scout/src/ScoutIndex.php(35): craft\elements\Entry::find()
#20 /Users/danielhowells/Sites/********/api/vendor/rias/craft-scout/src/ScoutIndex.php(40): rias\scout\ScoutIndex->__construct()
#21 /Users/danielhowells/Sites/********/api/config/scout.php(5): rias\scout\ScoutIndex::create()
#22 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/services/Config.php(211): craft\services\Config->unknown()
#23 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/services/Plugins.php(887): craft\services\Config->getConfigFromFile()
#24 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/services/Plugins.php(230): craft\services\Plugins->createPlugin()
#25 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/base/ApplicationTrait.php(1375): craft\services\Plugins->loadPlugins()
#26 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/web/Application.php(112): craft\web\Application->_postInit()
#27 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\web\Application->init()
#28 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/base/Application.php(206): craft\web\Application->__construct()
#29 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/src/web/Application.php(100): craft\web\Application->__construct()
#30 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/di/Container.php(400): craft\web\Application->__construct()
#31 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/di/Container.php(400): ReflectionClass->newInstanceArgs()
#32 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/di/Container.php(159): yii\di\Container->build()
#33 /Users/danielhowells/Sites/********/api/vendor/yiisoft/yii2/BaseYii.php(351): yii\di\Container->get()
#34 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/bootstrap/bootstrap.php(244): yii\BaseYii::createObject()
#35 /Users/danielhowells/Sites/********/api/vendor/craftcms/cms/bootstrap/web.php(51): ::unknown()
#36 /Users/danielhowells/Sites/********/api/web/index.php(20): ::unknown()
#37 {main}
2019-12-20 23:23:33 [-][-][-][info][application] $_GET = [
    'p' => 'graphql'
]

$_POST = []

$_FILES = []

$_COOKIE = []

$_SERVER = [
    'REDIRECT_STATUS' => '200'
    'HTTP_HOST' => 'api.********.test:8888'
    'HTTP_CONNECTION' => 'keep-alive'
    'HTTP_PRAGMA' => 'no-cache'
    'HTTP_CACHE_CONTROL' => 'no-cache'
    'HTTP_ACCESS_CONTROL_REQUEST_METHOD' => 'POST'
    'HTTP_ORIGIN' => 'http://********.test:8888'
    'HTTP_USER_AGENT' => 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.88 Safari/537.36'
    'HTTP_DNT' => '1'
    'HTTP_ACCESS_CONTROL_REQUEST_HEADERS' => 'authorization,content-type'
    'HTTP_ACCEPT' => '*/*'
    'HTTP_REFERER' => 'http://********.test:8888/'
    'HTTP_ACCEPT_ENCODING' => 'gzip, deflate'
    'HTTP_ACCEPT_LANGUAGE' => 'en-US,en;q=0.9,de;q=0.8,nl;q=0.7,ru;q=0.6,da;q=0.5,it;q=0.4,ja;q=0.3,la;q=0.2'
    'PATH' => '/usr/bin:/bin:/usr/sbin:/sbin'
    'SERVER_SIGNATURE' => ''
    'SERVER_SOFTWARE' => 'Apache'
    'SERVER_NAME' => 'api.********.test'
    'SERVER_ADDR' => '::1'
    'SERVER_PORT' => '8888'
    'REMOTE_ADDR' => '::1'
    'DOCUMENT_ROOT' => '/Users/danielhowells/Sites/********/api/web'
    'SERVER_ADMIN' => 'you@example.com'
    'SCRIPT_FILENAME' => '/Users/danielhowells/Sites/********/api/web/index.php'
    'REMOTE_PORT' => '61183'
    'REDIRECT_QUERY_STRING' => 'p=graphql'
    'REDIRECT_URL' => '/graphql'
    'GATEWAY_INTERFACE' => 'CGI/1.1'
    'SERVER_PROTOCOL' => 'HTTP/1.1'
    'REQUEST_METHOD' => 'OPTIONS'
    'QUERY_STRING' => 'p=graphql'
    'REQUEST_URI' => '/graphql'
    'SCRIPT_NAME' => '/index.php'
    'PHP_SELF' => '/index.php'
    'REQUEST_TIME_FLOAT' => 1576913013.612
    'REQUEST_TIME' => 1576913013
    'argv' => [
        0 => 'p=graphql'
    ]
    'argc' => 1
    'ENVIRONMENT' => 'dev'
    'SECURITY_KEY' => '••••••••••••••••••••••••••••••••'
    'DB_DRIVER' => 'mysql'
    'DB_SERVER' => 'localhost'
    'DB_USER' => '********'
    'DB_PASSWORD' => '••••••••••'
    'DB_DATABASE' => '********'
    'DB_SCHEMA' => 'public'
    'DB_TABLE_PREFIX' => ''
    'DB_PORT' => ''
    'BASE_URL' => 'http://api.********.test:8888'
    'DEFAULT_SITE_URL' => 'http://api.********.test:8888'
]
```

**Comment by brandonkelly** [comment]
Thanks. I see what’s going on, and just released 3.3.18.4 to fix it.

**Comment by howells** [comment]
Great, thanks - seems to be working fine now
